lib/page-object/platforms/selenium_webdriver.rb

```diff
@@ -9,7 +9,20 @@ def self.create_page_object(browser)
 
       def self.is_for?(browser)
         require 'selenium-webdriver'
-        browser.is_a? ::Selenium::WebDriver::Driver
+        browser.is_a?(::Selenium::WebDriver::Driver) || browser.is_a?(::Selenium::WebDriver::Element)
+      end
+
+      def self.browser_for root
+        return root if root.is_a?(::Selenium::WebDriver::Driver)
+        Selenium::WebDriver::Driver.new(root.send(:bridge))
```

Comment:
This takes an element and provides a webdriver from that element. It's a little messy, but it's the best implementation I could figure out at the time.